src/components/wallet/ActionAccessGate.tsx

```
@@ -0,0 +1,49 @@
+  fullWidth?: boolean
+}) {
+  const { isConnected, chain } = useAccount()
+
+  if (!isConnected) {
+    return (
+      <Box w={fullWidth ? "100%" : "auto"}>
+        <VisuallyHidden aria-live="polite">
+          Wallet not connected
+        </VisuallyHidden>
+        <ConnectButton
+          children={"Connect wallet to withdraw"}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)